basestar-database-server/src/test/java/io/basestar/database/TestDatabaseServer.java

```diff
@@ -1115,4 +1117,16 @@ void testMessage() {
         System.err.println(b64);
 
     }
+
+    @Test
```

Comment:
Test that shows a schema with a sequence associated gets an auto-incremented id instead of a UUID.